test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/AutoRenewEntitiesForReconnect.java

```diff
@@ -110,7 +110,7 @@ protected Logger getResultsLogger() {
 	 * Since reconnect is not supported when node starts from genesis, run some transactions before running correctness
 	 * tests so that a state is saved before reconnect.
 	 *
-	 * @return
+	 * @return HapiApiSpec needed to be run
```

Comment:
HapiApiSpec to do some CryptoTransfer's before reconnect

Reply:
fixed. Thanks @qnswirlds